### ISSUE TITLE
Letsencrypt error - Can't load /var/azuracast/.rnd into RNG

### ISSUE THREAD

**Issue by pwrj** [AzuraCast/AzuraCast#2527]
**I can no longer access my station. When I was setting up "Letsencrypt" I got the following error. Please help. Thanks**

IMPORTANT NOTES:
 - The following errors were reported by the server:

   Domain: stream.mydomain.com
   Type:   unauthorized
   Detail: Invalid response from
   https://stream.mydomain.com/.well-known/acme-challenge/WhI0GdF-6jERo- 
   yMEmgrdVr9AlpEUsKALH7SaUI_0Fc
   [123.456.789.10removed for privacy]: "<html>\r\n<head><title>404 Not
   Found</title></head>\r\n<body>\r\n<center><h1>404 Not
   Found</h1></center>\r\n<hr><center>nginx</center>\r\n"

   To fix these errors, please make sure that your domain name was
   entered correctly and the DNS A/AAAA record(s) for that domain
   contain(s) the right IP address.
Domain name stream.mydomain.com is not set up with LetsEncrypt yet. Reverting to self-signed cert...
Can't load /var/azuracast/.rnd into RNG
140635823501760:error:2406F079:random number generator:RAND_load_file:Cannot open file:../crypto/rand/randfile.c:88:Filename=/var/azuracast/.rnd
Generating a RSA private key
.......+++++
........................................................+++++
writing new private key to '/etc/letsencrypt/selfsigned.key'

**Comment by SlvrEagle23** [comment]
@pwrj You should still be able to access your station using the self-signed SSL certificate even if LetsEncrypt setup fails. If you've already set up the necessary domain name, it may just be that DNS hasn't propagated the changes yet, and Letsencrypt setup will work if attempted a little later.

**Comment by pwrj** [comment]
@SlvrEagle23 Thank you for your reply. How do, I access it. Domain and IP don't work.

**Comment by pwrj** [comment]
@SlvrEagle23 - Here's what I get, when I attempt to reach using the IP:10080

This site can’t be reached 123.456.789.10 refused to connect.
Try:

Checking the connection
Checking the proxy and the firewall
ERR_CONNECTION_REFUSED

**Comment by SlvrEagle23** [comment]
@pwrj You're using port 10080? You can't use Letsencrypt directly with AzuraCast if you're hosting it on port 10080; it has to use the standard 443/80 web ports for LetsEncrypt to work properly.

**Comment by pwrj** [comment]
@SlvrEagle23 I'm sorry, I hadn't realize that. What can I do to access the radio?

**Comment by SlvrEagle23** [comment]
@pwrj Your station should still be serving on whatever port you selected. You may need to check your logs with `docker-compose logs -f web` on the host computer.

**Comment by pwrj** [comment]
@SlvrEagle23 Surprisingly, the radio is running, I noticed this line:
web_1       | 2020/03/18 22:37:50 Problem with dial: dial tcp **_172.19.0.3_**:3306: connect: connection refused. Sleeping 1s

**Comment by SlvrEagle23** [comment]
@pwrj That's completely normal...check further down in the logs to see if nginx is reporting any errors and not starting up.

**Comment by pwrj** [comment]
@SlvrEagle23 Nothing relating to nginx at all. What am I missing?

**Comment by SlvrEagle23** [comment]
@pwrj I'm not sure...did you change the HTTP port to 10080? If so, then `https://that-port:10080` won't work, the `http://` has to match which port you're connecting to.

**Comment by pwrj** [comment]
@SlvrEagle23 I did. but I double check to make sure. .env file has:

COMPOSE_PROJECT_NAME=azuracast

AZURACAST_HTTP_PORT=10080
AZURACAST_HTTPS_PORT=10443

AZURACAST_FTP_PORT=31
AZURACAST_FTP_PASV_RANGE=30000-30009

**Comment by SlvrEagle23** [comment]
@pwrj That's fine, so `https://your-ip:10443` isn't working either?

**Comment by pwrj** [comment]
@SlvrEagle23 Nope. I get:

Your connection is not private
Attackers might be trying to steal your information from IP (for example, passwords, messages, or credit cards). Learn more
NET::ERR_CERT_INVALID

Help improve Chrome security by sending URLs of some pages you visit, limited system information, and some page content to Google. Privacy policy
IP normally uses encryption to protect your information. When Google Chrome tried to connect to IP this time, the website sent back unusual and incorrect credentials. This may happen when an attacker is trying to pretend to be IP or a Wi-Fi sign-in screen has interrupted the connection. Your information is still secure because Google Chrome stopped the connection before any data was exchanged.

You cannot visit IP right now because the website sent scrambled credentials that Google Chrome cannot process. Network errors and attacks are usually temporary, so this page will probably work later.

**Comment by SlvrEagle23** [comment]
@pwrj No, that's ok, just click through that (you may have to click "more info" or something, then allow it)...that's just the self-signed cert warning that you see when LetsEncrypt isn't set up.

You can then go into the system settings, disable "Force HTTPS", then you can use 10080 if you want.

**Comment by pwrj** [comment]
@SlvrEagle23 No, I don't have the option to move forward unfortunately. 

You cannot visit IP right now because the website sent scrambled credentials that Google Chrome cannot process. Network errors and attacks are usually temporary, so this page will probably work later.

But, you got me thinking, so I used a browser other than Chrome and I did have the option to forward and eventually logged in. Wow, interesting.  Thank you my friend. You saved me a reinstall and lots of headaches. Thank you thank you!!!
